packages/cli/src/build/build.ts

```diff
@@ -68,7 +68,8 @@ const getOptions = (config?: MitosisConfig): MitosisConfig => ({
 });
 
 async function clean(options: MitosisConfig) {
-  const files = await glob(`${options.dest}/**/*/${options.files}`);
+  const patterns = options.targets.map((target) => `${options.dest}/${target}/*/${options.files}`);
```

Comment:
We should use `getTargetPath` instead of `${target}`:

```suggestion
  const patterns = options.targets.map((target) => `${options.dest}/${options.getTargetPath(target)}/${options.files}`);
```